src/erlfmt.erl

```diff
@@ -498,9 +498,9 @@ check_line_lengths(FileName, Width, String, {FirstLineNo, _}) ->
     LastLineNo = FirstLineNo + length(Lines) - 1,
     [
         {FileName, LineNo, ?MODULE, {long_line, string:length(Line), Width}}
-     || {LineNo, Line} <- lists:zip(
-            lists:seq(FirstLineNo, LastLineNo),
-            Lines
-        ),
-        string:length(Line) > Width
+        || {LineNo, Line} <- lists:zip(
```

Comment:
This is an old format

Reply:
Thank you for the heads up, this must be the issue, somehow locally I'm running with an old formatter, will try to figure it out.

Reply:
turns out I had an `ebin` dir in the root of my repo with old compiled beam files. After removing all is well. Sorry about the noise.

Reply:
This happens to me so much, no worries.
I run `rm -rf _build` probably much more than is required.